test-plans/ping/src/main.rs

```diff
@@ -0,0 +1,108 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use futures::StreamExt;
+use libp2p::swarm::{keep_alive, SwarmEvent, NetworkBehaviour};
+use libp2p::*;
+use std::collections::HashSet;
+use std::time::Duration;
+use ::ping::{run_ping, PingSwarm};
+use ursa_network::{NetworkConfig, Behaviour as UrsaBehaviour};
+use libipld::DefaultParams;
+use ursa_store::{BitswapStorage, Store};
+use db::{rocks::RocksDb, rocks_config::RocksDbConfig};
+use std::sync::Arc;
+
+#[async_std::main]
+async fn main() -> Result<()> {
+    let local_key = identity::Keypair::generate_ed25519();
+    let local_peer_id = PeerId::from(local_key.public());
+
+    //
+    let db = RocksDb::open("test_db", &RocksDbConfig::default())
```

Comment:
just wanted to see if we can use memorydb instead rocksdb for tests?  we probably don't need to persist the data for test plans. I started using that in our unit test as well.
not sure if it directly applies but something like this:
https://github.com/fleek-network/ursa/blob/a38223782ed2b66b90930332d7cf8cbb2169e7d1/crates/ursa-network/src/service.rs#L880

Reply:
Yes I agreed. Will use memorydb for test plans.